presto-main/src/main/java/com/facebook/presto/sql/planner/Partitioning.java

```diff
@@ -87,6 +87,11 @@ public PartitioningHandle getHandle()
         return arguments;
     }
 
+    public boolean isOnlyPartitionedOnColumns()
```

Comment:
This name is very very confusing - somebody would look at this and say 'on which columns'?

Reply:
Also, 'column' has no specific meaning - a constant is also a column. I think you want to be specific about symbol references.

Reply:
it's corresponding to the `getColumns` function next to this.